src/Package/Impl/ProjectSystem/ProjectSystemServices.cs

```diff
@@ -28,7 +29,11 @@ internal sealed class ProjectSystemServices : IProjectSystemServices {
             DTE dte = VsAppShell.Current.GetGlobalService<DTE>();
             if (dte.Solution.Projects.Count > 0) {
                 try {
-                    return dte.Solution?.Projects?.Cast<EnvDTE.Project>()?.First();
+                    var projects = dte.Solution?.SolutionBuild?.StartupProjects as IEnumerable;
```

Comment:
From https://msdn.microsoft.com/en-us/library/envdte.solutionbuild.startupprojects.aspx:
```cs
        SolutionBuild sb = dte.Solution.SolutionBuild;
        SolutionConfiguration sc = sb.ActiveConfiguration;
        vsBuildState vsBS;
        string msg = "Return relative path to startup projects: ";
        foreach (String s in (Array)sb.StartupProjects) {
            msg += "\n   " + s;
        }
```
So it is an array. In this case,
```cs
var projectName = (dte.Solution?.SolutionBuild?.StartupProjects as IEnumerable)?.Cast<string>().FirstOrDefault();
```